JUnitParamsAndMockito_task7/src/main/java/org/Marta/TrainingEvaluator.java

```diff
@@ -0,0 +1,58 @@
+package org.Marta;
+
+public class TrainingEvaluator {
+    private Training training;
+    private static final int LENGTH_WEIGHT = 1;
+    private static final int KCAL_WEIGHT = 2;
+    private static final int PULSE_WEIGHT = 3;
+
+    public TrainingEvaluator(Training training) {
+        this.training = training;
+    }
+
+    public int rateForLength() {
+        if (training == null || training.getLengthInMinutes() <= 0 || training.getLengthInMinutes() > 720) {
+            throw new IllegalLength("The length of training should be in the range of 1 - 720 minutes");
+        } else if (training.getLengthInMinutes() < 30) {
+            return 1;
+        } else if (training.getLengthInMinutes() <= 60) {
+            return 2;
+        } else {
+            return 3;
```

Comment:
Tutaj bym zaproponował jakiś enum, bo co znaczy "1"/"2" itd? Może lepiej gdyby to było coś w stylu "HIGH" "LOW" "MEDIUM" ;)